cli/man/transact-playlist.1.md

```diff
@@ -0,0 +1,170 @@
+% TRANSACT-PLAYLIST(1) Cargill, Incorporated | Transact Commands
```

Comment:
This file looks like it's two separate man pages?

Reply:
whoops, somehow managed to copy the workload one into this one

Reply:
fixed